doc/MANUAL.adoc

```diff
@@ -918,6 +918,11 @@ Examples:
 *time_macros*::
     Ignore `+__DATE__+`, `+__TIME__+` and `+__TIMESTAMP__+` being present in the
     source code.
+*full_command_line*::
```

Comment:
I think that this name could be improved. Naming of sloppiness options should follow "I'm OK if ccache is sloppy about \<name>", so perhaps `record_gcc_switches`?

Also, please keep the list of sloppiness names sorted alphabetically (also in other places).